components/MapView.js

```diff
@@ -14,9 +14,15 @@ import MapPolyline from './MapPolyline';
 import MapPolygon from './MapPolygon';
 import MapCircle from './MapCircle';
 import MapCallout from './MapCallout';
+import {
+  contextTypes,
+  airMapName,
+  AIRGoogleMapIsInstalled,
```

Comment:
Can we do this before merging? I don't think `AIR` prefixes should really make it to JS user-land code...